prismlauncher

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# discord RPC
+for i in {0..9}; do
+    test -S $XDG_RUNTIME_DIR/discord-ipc-$i || ln -sf {app/com.discordapp.Discord,$XDG_RUNTIME_DIR}/discord-ipc-$i;
```

Comment:
Shouldn't `$XDG_RUNTIME_DIR` be quoted here? AFAIK it's allowed to contain spaces.

Reply:
good find!